### ISSUE TITLE
Docs: off-canvas example focus order is incorrect

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#22847]
In the off-canvas example, once the extra nav is collapsed, keyboard users can toggle the navigation - but then have to navigate through the rest of the page before reaching the off-canvas navigation.

![offcanvas-focus](https://user-images.githubusercontent.com/895831/27252738-c54be95c-535d-11e7-9d52-e984b8cb2416.PNG)

Ideally, once the off-canvas navigation is toggled, focus should be moved to it programmatically.
